web/src/compose_reply.js

```diff
@@ -64,7 +71,10 @@ export function respond_to_message(opts) {
             const first_operand = first_term.operand;
 
             if (first_operator === "stream" && !stream_data.is_subscribed_by_name(first_operand)) {
-                compose_actions.start("stream", {trigger: "empty_narrow_compose"});
+                compose_actions.start("stream", {
+                    trigger: "empty_narrow_compose",
+                    keep_composebox_empty: opts.keep_composebox_empty,
+                });
                 return;
             }
```

Comment:
I don't understand this case at all - the existing logic seems crazy. I don't see why we're looking at the first term rather than just the `stream` operand, if any. But maybe it's a case we'll delete in https://github.com/zulip/zulip/pull/21852?